UI/MemStickScreen.cpp

```diff
@@ -510,12 +525,17 @@ UI::EventReturn ConfirmMemstickMoveScreen::OnConfirm(UI::EventParams &params) {
 				std::string error = "Failed to read old directory";
 				INFO_LOG(SYSTEM, "%s", error.c_str());
 				progressReporter_.Set(iz->T(error.c_str()));
-				return new bool(false);
+				return new MoveResult{ false, error };
 			}
 
 			bool dryRun = false;  // Useful for debugging.
 
-			size_t moveFailures = 0;
+			size_t failedFiles = 0;
+			size_t skippedFiles = 0;
+
+			// We're not moving huge files like ISOs during this process, unless
+			// they can be directly moved, without rewriting the file.
+			const uint64_t BIG_FILE_THRESHOLD = 24 * 1024 * 1024;
```

Comment:
Hm, install and saved replay movie data in some games might exceed this and confuse users if not moved.

-[Unknown]

Reply:
Maybe should display a list of not-moved files? Not sure.

Reply:
Hm, maybe.  I guess as long as it says some larger files weren't moved... I wonder if some people might not even be able to access internal storage though?  Then again, how did the large files get there, then?  So probably okay...

-[Unknown]